core_book/Feats.tex

```diff
@@ -521,7 +521,7 @@ \section{Feat Tables}
         \ff[8]{Greater Specialization} The bonus from your \textit{specialization} ability increases to \plus4.
 
         \ff[11]{Item Attunement} You gain an additional \glossterm{item slot}.
-        In addition, crafting magic items takes you one hour per 100 gp of material components.
+        In addition, crafting magic items takes you one hour per 1,000 gp of material components.
```

Comment:
This would be a weird scaling, since this is a 10x increase and the 17th level ability would be a 2.5x increase. My original intention was for both this ability and the 17th level ability to be a 5x increase:
```suggestion
        In addition, crafting magic items takes you one hour per 500 gp of material components.
```